packages/allure-playwright/src/helpers.ts

```diff
@@ -0,0 +1,105 @@
+import { TestInfo } from "@playwright/test";
+import { Label, LabelName, LinkType } from "allure-js-commons";
+
+export const ContentTypes = {
+  label: "allure/label",
+  link: "allure/link",
+};
+
+export function label(testInfo: TestInfo, label: Label) {
+  testInfo.attachments.push({
+    name: label.name,
+    contentType: ContentTypes.label,
+    path: label.value,
+  });
+}
+
+export function epic(testInfo: TestInfo, epic: string) {
+  label(testInfo, {
+    name: LabelName.EPIC,
+    value: epic,
+  });
+}
+
+export function feature(testInfo: TestInfo, feature: string) {
+  label(testInfo, {
+    name: LabelName.FEATURE,
+    value: feature,
+  });
+}
+
+export function story(testInfo: TestInfo, story: string): void {
+  label(testInfo, {
+    name: LabelName.STORY,
+    value: story,
+  });
+}
+
+export function suite(testInfo: TestInfo, name: string): void {
+  label(testInfo, {
+    name: LabelName.SUITE,
+    value: name,
+  });
+}
+
+export function parentSuite(testInfo: TestInfo, name: string) {
+  label(testInfo, {
+    name: LabelName.PARENT_SUITE,
+    value: name,
+  });
+}
+
+export function subSuite(testInfo: TestInfo, name: string) {
+  label(testInfo, {
+    name: LabelName.SUB_SUITE,
+    value: name,
+  });
+}
+
+export function owner(testInfo: TestInfo, owner: string) {
+  label(testInfo, {
+    name: LabelName.OWNER,
+    value: owner,
+  });
+}
+
+export function severity(testInfo: TestInfo, severity: string) {
+  label(testInfo, {
+    name: LabelName.SEVERITY,
+    value: severity,
+  });
+}
+
+export function tag(testInfo: TestInfo, tag: string) {
+  label(testInfo, {
+    name: LabelName.SEVERITY,
```

Comment:
Did you mean LabelName.TAG here? (is it a thing?)

Reply:
https://github.com/allure-framework/allure-js/blob/master/packages/allure-js-commons/src/model.ts#L117